spec/features/create_extension_spec.rb

```diff
@@ -101,7 +101,7 @@ def check_run_specs
 
   def check_sandbox
     cd(install_path) do
-      command = 'bin/rails runner "puts %{The version of SolidusTestExtension is #{SolidusTestExtension::VERSION}}"'
+      command = 'bin/test_app runner "puts %{The version of SolidusTestExtension is #{SolidusTestExtension::VERSION}}"'
```

Comment:
This should be something other than "test_app", the sandbox was introduced precisely to avoid using the test_app for direct development and experimentation, leaving it just as a test dummy. So `bin/dev_app` or `bin/sandbox-rails` might be better names.